Src/BootCamp.Chapter/Program.cs

```diff
@@ -6,7 +6,45 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            string name;
+            string surename;
+            int age;
+            double weight;
+            double height;
+
+            GetPersonData();
+
+            double BMI;
+            CalculateBMI(weight, height);
+
+            Console.WriteLine($"{name} {surename} is {age}, weight is {weight} kg and his height is {height} cm.");
+            Console.WriteLine($"BMI is {BMI:f2}");
+
+
+
+
+            void GetPersonData()
```

Comment:
For now, I recommend breaking this down into many smaller functions each returning a different thing: name, surename, age, weight, height.